### PR TITLE
Combined Articles

### PR DESCRIPTION
**Pull request by Compositr**
As requested @Katloo24, this PR adds the ability to combine multiple articles into one. Provides users with the ability to cycle through sidebar options on these combined articles.

Todo: 
- [x] Refactor article rendering
- [x] Implement combined articles functionality
- [x] Apply refactor to all articles

### CODE REVIEW

**Review by Compositr — COMMENTED** [review]
Todo: 
- [ ] Refactor article rendering
- [ ] Implement combined articles functionality
- [ ] Apply refactor to all articles

**Comment by kenos1** [comment]
How would a wiki maintainer be able to take advantage of this feature?

File: app/(wiki)/(articles)/obstacles/[item]/page.tsx [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+    }
+  ]
+});
 
-export default GenericMDXPageFactory({
-  path: "obstacles"
-})
\ No newline at end of file
+exports = toExport;
+export default toExport.default;
```
Comment (Compositr) [inline app/(wiki)/(articles)/obstacles/[item]/page.tsx:33]: This is an example of how the rendering refactor works.
You can specify which items get combined into one article (like all the crates, trees, etc.). Regex soon:tm:

Comment (Compositr) [reply app/(wiki)/(articles)/obstacles/[item]/page.tsx:33]: @kenos1 Basically it allows wiki maintainers to merge very similar articles at their discretion.

Comment (kenos1) [reply app/(wiki)/(articles)/obstacles/[item]/page.tsx:33]: Awesome, lgtm

**Comment by kenos1** [comment]
LGTM, @Katloo24 @hsanger ?

**Comment by Compositr** [comment]
meh imma merge it cause i wanna start writing

**Pull request merged**